Sources/main.cpp

```diff
@@ -109,6 +215,8 @@ void PrintUsage(const char *programName)
             "\n"
             "Options:\n"
             "  -c pfx-file   sign the APPX with the private key file\n"
+            "  -f map-file   specify inputs from a mapping file\n"
```

Comment:
It popped into my head after reading

    MakeAppx pack /f mapping_filepath /p filepath.appx

I can change it if you'd like. I have no preference. (I'd rather kill all the short options and only have long options. =])